server/dash/dashboards/neighborhood_recent.py

```diff
@@ -209,12 +199,16 @@ def update_figure(selected_council):
 
 
 @app.callback(
-    Output("pie_graph", "pie_fig"),
+    Output("pie_graph", "figure"),
     Input("council_list", "value")
 )
 def update_council_figure(selected_council):
     pie_df = df.query(f"councilName == '{selected_council}' and createdDate >= '{start_date}'").groupby(['typeName']).agg('count').reset_index()  # noqa
-
+    if pie_df.shape[0] == 0:
+        pie_df = pd.DataFrame(columns=["srnumber", "typeName"])
+        pie_df.loc[0] = [12345678, "No Request at this time"]
```

Comment:
I think it makes more sense to use 0 here, since there are no requests.

Reply:
Set to 0. I wasn't sure if the pie chart would show anything if I set it to 0.